plugins/inputs/docker/README.md

```diff
@@ -142,6 +142,23 @@ some storage drivers such as devicemapper.
     - total
     - used
 
+The above measurements for the devicemapper storage driver can now be found in the new `docker_devicemapper` measurement
+
+- docker_devicemapper
```

Comment:
You don't need to fix this, but this is good opportunity to show an obscure README issue.  In Markdown, or maybe it's only Github Markdown, if you have multiple nested lists you will need to alter the first list character when repeating them, otherwise you get a weird bit of inconsistent vertical whitespace.  To see this, in the current plugin README compare the vertical whitespace difference between the docker measurement and the docker_data.  We usually use `-` and `+`.

Bad:
```
- foo
  - bar
- foo2
  - bar2
- foo3
  - bar3
- foo4
  - bar4
```
Good:
```
- foo
  - bar
+ foo2
  - bar2
- foo3
  - bar3
+ foo4
  - bar4
```

Reply:
Oh yeah that is weird. Nice I did not know this!

Reply:
What have you done. Now I can't unsee the whitespace 😂 :trollface: